docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,442 @@ card(
     />
-  );
-}
-    `}
-  />,
+  </MainSection>,
 );
 
 card(
-  <Example
-    id="helperText"
-    name="Example: Helper Text"
-    description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
-    defaultCode={`
+  <MainSection name="Variants">
+    <MainSection.Subsection title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="Large"
+        description={`Use \`lg\` as the default size within Pinterest products.`}
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Algeria', value: 'algeria'},
+    {label:'Belgium', value: 'belgium'},
+    {label:'Canada', value: 'canada'},
+    {label:'Denmark', value: 'denmark'},
+    {label:'Egypt', value: 'egypt'},
+    {label:'France', value: 'france'},
+  ]}
+  size='lg'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="Medium"
+        description={`Use \`md\` on denser surfaces, such as Business products or internal tools.`}
+        defaultCode={`
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Algeria', value: 'algeria'},
+      {label:'Belgium', value: 'belgium'},
+      {label:'Canada', value: 'canada'},
+      {label:'Denmark', value: 'denmark'},
+      {label:'Egypt', value: 'egypt'},
+      {label:'France', value: 'france'},
+    ]}
+    size='md'
+  />`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Helper text"
+      description="Helper text should be used when additional description may be required to understand the SelectList. Most commonly, this could be text that is legally required to be displayed, or instructions to fill out a form (e.g. proper formatting). If the text is optional, Tooltip could be used instead."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label: 'ARS - Argentine peso', value: 'ars'},
+    {label: 'AUD - Australian dollar', value: 'aud'},
+    {label: 'ERN - Eritrean nakfa', value: 'ern'},
+    {label: 'EUR - Euro', value: 'eur'},
+    {label: 'GBP - British pound', value: 'gbp'},
+    {label: 'JPY - Japanese yen', value: 'jpy'},
+    {label: 'USD - United States Dollar', value: 'usd'},
+  ]}
+  label='Default currency'
+  helperText='Product prices in your data source without an ISO currency code will default to this currency'
+  size='lg'
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Controlled component"
+      description="The controlled variant allows form data to be handled by the component. You will ned to use this pattern in situations where you want to explicitly set `value` or `placeholder` props."
```

Comment:
"SelectList must be used as a controlled component when the `placeholder` or `value` props are needed. When used in this manner, `onChange` and `value` are required, while `placeholder` is optional."

Reply:
Great suggestion. Thank you!